cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1540,4 +1618,39 @@ pub(super) mod test {
         PackageToInstall::from_str("@1.2.3").expect_err("missing attrpath should cause failure");
         PackageToInstall::from_str("foo@").expect_err("missing version should cause failure");
     }
+
+    proptest! {
+        #[test]
+        fn manifest_round_trip(manifest in any::<TypedManifestCatalog>()) {
+            let toml = toml_edit::ser::to_string(&manifest).unwrap();
+            let parsed = toml_edit::de::from_str::<TypedManifestCatalog>(&toml).unwrap();
+            prop_assert_eq!(manifest, parsed);
+        }
+    }
+
+    #[test]
+    fn parses_services_section() {
+        let manifest_raw = indoc! {r#"
+        version = 1
+
+        [services.foo]
+        command = "start foo"
+        [services.foo.vars]
+        bar = "qux"
+        "#};
+        let TypedManifest::Catalog(parsed) =
+            toml_edit::de::from_str::<TypedManifest>(manifest_raw).unwrap()
+        else {
+            panic!("failed to parse manifest");
+        };
+        assert_eq!(
+            parsed
+                .services
+                .get("foo")
+                .and_then(|foo| foo.vars.as_ref())
+                .and_then(|vars| vars.0.get("bar"))
+                .unwrap(),
+            "qux"
+        );
+    }
```

Comment:
nonblocking: I'm not sure adding tests for this is really worthwhile. I'm sure we'll have lots of other things that depend on this

Reply:
Fair, plus it's being exercised a bit by proptest.

Reply:
Fixed